static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/variant.js

```diff
@@ -42,7 +42,7 @@ const htmlTemplate: ({
             <a class="signin-gate__dismiss js-signin-gate__dismiss" href="#maincontent">Not Now</a>
         </div>
         <div class="signin-gate__padding-bottom signin-gate__buttons">
-            Already registered, contributed, or subscribed?&nbsp;<a class="signin-gate__link js-signin-gate__sign-in" href="${signInUrl}">Sign in</a>
+            Already registered, contributed, or subscribed?&nbsp;<a class="signin-gate__link js-signin-gate__sign-in signin-gate__link-no-ptm signin-gate__center-424" href="${signInUrl}">Sign in</a>
```

Comment:
Does this need a new class name to achieve this styling ? Can't it just be added to `signin-gate__link`

Reply:
we don't want to center all the links, that's why i added a separate class

Reply:
👍